docker/development/docker-compose.yml

```diff
@@ -54,6 +56,28 @@ services:
       <<: *x-email-defaults
       RAILS_ENV: development
       POSTGRES_HOST: db
+      WEBPACKER_DEV_SERVER_HOST: webpacker
+      WEBPACKER_DEV_SERVER_PORT: *WEBPACKER_DEV_SERVER_PORT
```

Comment:
These take precedence over the config in `config/webpacker.yml` and - when running via docker - tell the Rails application to use Dockers internal name resolution to find the `webpack-dev-server`. Essentially "Hey buddy, `webpack-dev-server` is running over here, not on `localhost` like it says in the `webpacker.yml`"